src/chocolatey/infrastructure.app/nuget/NugetList.cs

```diff
@@ -31,7 +31,8 @@ public static IEnumerable<IPackage> GetPackages(ChocolateyConfiguration configur
 
             if (configuration.AllVersions)
             {
-                return results.Where(PackageExtensions.IsListed).OrderBy(p => p.Id).ToList();
+                // just trust the server order, don't sort because that's a blocking operation
+                return results.Where(PackageExtensions.IsListed);

```

Comment:
Well, to be honest, Chocolatey and Nuget. However, the determination I made is that it's better to output in "source order" fast than to output in "Id Order" much, much, slower (not to mention memory impact). 

Point being that if we _do_ need to sort for certain sources,  we should do so MUCH further back up the stack (e.g. in the apps, where we enumerate the collection, not in the core library).